Misc/NEWS.d/next/Documentation/2024-01-28-14-31-53.gh-issue-114648.LkD8IM.rst

```diff
@@ -0,0 +1 @@
+Added :exc:`IndexError` to :meth:`pop` of :class:`list`.
```

Comment:
To prevent a Sphinx reference warning:
```suggestion
Added :exc:`IndexError` to :meth:`!pop` of :class:`list`.
```

But we don't usually need changelog NEWS entries for small docs changes, so shall we remove this file?

Reply:
Yes, please--I thought it was mandatory.
Not sure if I need to delete the file in a fresh commit, or if something else is required?

Reply:
> Yes, please--I thought it was mandatory. Not sure if I need to delete the file in a fresh commit, or if something else is required?

Yes, you can just delete a file in a new commit.